flutter/Graminsta/lib/auth/login.dart

```diff
@@ -34,17 +71,19 @@ class _LoginPageState extends State<LoginPage> {
           children: <Widget>[
             Flexible(
               child: TextFormField(
+                controller: _usernameController,
                 decoration: InputDecoration(
                   icon: Icon(
-                    Icons.email,
+                    Icons.person,
                   ),
-                  hintText: "Email Address",
+                  hintText: "Username",
                 ),
                 style: TextStyle(fontSize: 16, color: Colors.black),
               ),
```

Comment:
Maybe we need to add a `validator` to ensure the field is not null? 
https://flutter.dev/docs/cookbook/forms/validation